modules/core/module.go

```diff
@@ -175,11 +174,6 @@ func (AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedP
 	return nil
```

Comment:
You can delete this of all modules if you don't use it.
And as mentioned [here](https://github.com/cosmos/ibc-go/pull/3107#discussion_r1097611389), if you keep it, it should be still returning `WeightedProposalContent`.

Reply:
We will delete it as it is unused :+1: Thanks for reviewing!